pkg/regionpicker/cache.go

```diff
@@ -0,0 +1,119 @@
+package regionpicker
```

Comment:
```suggestion
package region
```

> It's helpful if everyone using the package can use the same name to refer to its contents, which implies that the package name should be good: short, concise, evocative. By convention, packages are given lower case, single-word names

https://golang.org/doc/effective_go#package-names

Reply:
i like `regionpicker` 😢

Reply:
I think `region.Pick` reads a lot better than `regionpicker.FindBest`.   I mean, why would we want to pick anything but the best?  Though I'd probably have chosen `region.Guess` if there is any likelihood of it being wrong or failing.

Reply:
Also, `region.Best` and `region.Current` work reasonably well too (don't need to use a verb if we think of it as a getter).